client/app/dedicatedCloud/datacenter/drp/index.js

```diff
@@ -0,0 +1,41 @@
+
+import DedicatedCloudDatacenterDrpCtrl from './dedicatedCloud-datacenter-drp.controller';
+import DedicatedCloudDatacenterDrpMainPccStepCtrl from './steps/common/mainPcc/dedicatedCloud-datacenter-drp-mainPccStep.controller';
+import DedicatedCloudDatacenterDrpOvhSecondPccStepCtrl from './steps/ovh/secondPcc/dedicatedCloud-datacenter-drp-ovh-secondPccStep.controller';
+import DedicatedCloudDatacenterDrpConfirmationStepCtrl from './steps/common/confirmation/dedicatedCloud-datacenter-drp-confirmationStep.controller';
+import DedicatedCloudDatacenterDrpConfirmationStepDeleteCtrl from './steps/common/confirmation/delete/dedicatedCloud-datacenter-drp-confirmationStep-delete.controller';
+
+import DedicatedCloudDrp from './dedicatedCloud-datacenter-drp.service';
+
+import {
+  DEDICATEDCLOUD_DATACENTER_DRP_IP_USAGE_MAC_ADDRESS_REG_EXP,
+  DEDICATEDCLOUD_DATACENTER_DRP_OPTIONS,
+  DEDICATEDCLOUD_DATACENTER_DRP_ORDER_OPTIONS,
+  DEDICATEDCLOUD_DATACENTER_DRP_ROLES,
+  DEDICATEDCLOUD_DATACENTER_DRP_STATUS,
+  DEDICATEDCLOUD_DATACENTER_DRP_UNAVAILABLE_IP_STATUS,
+  DEDICATEDCLOUD_DATACENTER_ZERTO,
+} from './dedicatedCloud-datacenter-drp.constants';
+
+angular.module('App')
+  .constant('DEDICATEDCLOUD_DATACENTER_DRP_IP_USAGE_MAC_ADDRESS_REG_EXP', DEDICATEDCLOUD_DATACENTER_DRP_IP_USAGE_MAC_ADDRESS_REG_EXP)
+  .constant('DEDICATEDCLOUD_DATACENTER_DRP_OPTIONS', DEDICATEDCLOUD_DATACENTER_DRP_OPTIONS)
+  .constant('DEDICATEDCLOUD_DATACENTER_DRP_ORDER_OPTIONS', DEDICATEDCLOUD_DATACENTER_DRP_ORDER_OPTIONS)
+  .constant('DEDICATEDCLOUD_DATACENTER_DRP_ROLES', DEDICATEDCLOUD_DATACENTER_DRP_ROLES)
+  .constant('DEDICATEDCLOUD_DATACENTER_DRP_STATUS', DEDICATEDCLOUD_DATACENTER_DRP_STATUS)
+  .constant('DEDICATEDCLOUD_DATACENTER_DRP_UNAVAILABLE_IP_STATUS', DEDICATEDCLOUD_DATACENTER_DRP_UNAVAILABLE_IP_STATUS)
+  .constant('DEDICATEDCLOUD_DATACENTER_ZERTO', DEDICATEDCLOUD_DATACENTER_ZERTO)
+  .controller('DedicatedCloudDatacenterDrpCtrl', DedicatedCloudDatacenterDrpCtrl)
+  .controller('DedicatedCloudDatacenterDrpMainPccStepCtrl', DedicatedCloudDatacenterDrpMainPccStepCtrl)
+  .controller('DedicatedCloudDatacenterDrpOvhSecondPccStepCtrl', DedicatedCloudDatacenterDrpOvhSecondPccStepCtrl)
+  .controller('DedicatedCloudDatacenterDrpConfirmationStepCtrl', DedicatedCloudDatacenterDrpConfirmationStepCtrl)
+  .controller('DedicatedCloudDatacenterDrpConfirmationStepDeleteCtrl', DedicatedCloudDatacenterDrpConfirmationStepDeleteCtrl)
+  .component('dedicatedCloudDatacenterDrp', dedicatedCloudDatacenterDrp)
+  .component('dedicatedCloudDatacenterDrpOvhMainPccStep', dedicatedCloudDatacenterDrpOvhMainPccStep)
+  .component('dedicatedCloudDatacenterDrpOvhSecondPccStep', dedicatedCloudDatacenterDrpOvhSecondPccStep)
+  .component('dedicatedCloudDatacenterDrpOvhConfirmationStep', dedicatedCloudDatacenterDrpOvhConfirmationStep)
+  .service('DedicatedCloudDrp', DedicatedCloudDrp);
```

Comment:
You probably should create submodules for each subfolder and import them here instead if it make sense